app/include/zmk/endpoints.h

```diff
@@ -33,7 +33,7 @@
  * Note that this value may change between firmware versions, so it should not
  * be used in any persistent storage.
  */
-#define ZMK_ENDPOINT_COUNT (ZMK_ENDPOINT_USB_COUNT + ZMK_ENDPOINT_BLE_COUNT)
+#define ZMK_ENDPOINT_COUNT (ZMK_ENDPOINT_USB_COUNT + ZMK_ENDPOINT_BLE_COUNT + 1)
```

Comment:
I think it would be good to use

```c
#define ZMK_ENDPOINT_NONE_COUNT 1
...
#define ZMK_ENDPOINT_COUNT (ZMK_ENDPOINT_NONE_COUNT + ZMK_ENDPOINT_USB_COUNT + ZMK_ENDPOINT_BLE_COUNT)
```
since otherwise it's not obvious what the `+ 1` is for.